src/main/cpp/RobotContainer.cpp

```diff
@@ -14,27 +10,45 @@ RobotContainer::RobotContainer()
   , m_coral_intake()
   , m_climber()
   , m_elevator()
+  , m_coordinator(&m_elevator, &m_coral_intake)
   , m_auto_leave(swerve_drive, 0_in, -3_ft, 0_deg)
 {
   ConfigureBindings();
 }
 
 void RobotContainer::ConfigureBindings() 
 {
-  ctrl->RightTrigger(0.75).WhileTrue(m_climber.RunArm(0.1));
-  ctrl->LeftTrigger(0.75).WhileTrue(m_climber.RunArm(-0.1));
-  ctrl->LeftBumper().OnTrue(m_algae_intake.MoveWristByPowerCommand(-0.5));
+  ctrl->RightTrigger(0.75).OnTrue(m_climber.Climb());
+
+  ctrl->LeftTrigger(0.75).OnTrue(m_elevator.HalfSpeed()
+  .AlongWith(swerve_drive->ToggleFarisModeCommand()));
```

Comment:
The indentation is a little funky here. Suggest something like this:

```suggestion
  ctrl->LeftTrigger(0.75)
    .OnTrue(m_elevator.HalfSpeed().AlongWith(swerve_drive->ToggleFarisModeCommand()));
```

… or perhaps …

```suggestion
  ctrl->LeftTrigger(0.75)
    .OnTrue(m_elevator.HalfSpeed()
              .AlongWith(swerve_drive->ToggleFarisModeCommand()));
```